pandas/plotting/_core.py

```diff
@@ -2738,18 +2738,81 @@ def bar(self, x=None, y=None, **kwds):
 
     def barh(self, x=None, y=None, **kwds):
         """
-        Horizontal bar plot
+        Make a horizontal bar plot.
+
+        A horizontal bar plot is a plot that presents categorical data with
```

Comment:
Actually it's not categorical data (categories like male and female) but quantitative data (numbers like age or income).